agreed/0004-Farm-attempt/0004-Farm-attempt.md

```diff
@@ -32,22 +32,19 @@ ON receipt of a `Get` request for `ImmutableData` the `DataManager` calculates t
 1. PmidNodes -> Registers an Optional (if set by user) wallet address on any store of data to it
 2. App Developer -> App developers will include wallet address in any `Get` request
 3. Publisher -> As data is stored a wallet address of the owner is stored if this is first time seen on the network (stored in DM account for the data element)
-4. Core development -> Initially every node will be aware of a hard coded wallet address for core development. This will likely lead to a multisig wallet.
+4. Core development -> Initially every node will be aware of a hard coded wallet address for core development. This will likely lead to a multi-sign wallet.
```

Comment:
Yes, "multisig wallet" is definitely more common than "multi-sign". The other option is to say "multisignature wallet".

https://en.wikipedia.org/wiki/Multisignature